index.js

```diff
@@ -134,6 +134,8 @@ module.exports = {
     "spaced-comment": ["warn", "always", { "exceptions": ["-", "+"] }],
     'no-console': 'warn',
     'default-param-last': 'warn',
+    "curly": ['warn', "all"],
```

Comment:
This is to add `{}` for if statement.

```ts
if (conditionA) return /// warn

if(conditionA) {
  return
}
```

ref: https://eslint.org/docs/latest/rules/curly#rule-details